app/views/admin/content_pages/preview_of_live.erb

```diff
@@ -0,0 +1,14 @@
+<strong class="govuk-tag">
+  Preview
+</strong>
```

Comment:
Again - "Preview" looks like a heading. This HTML should really have some context.